Makefile

```diff
@@ -96,6 +96,9 @@ init:
 	sysrepoctl -s $(GS_YANG_REPO) --install $(GS_YANG_REPO)/goldstone-onlp.yang
 	sysrepoctl -s $(GS_YANG_REPO) --install $(GS_YANG_REPO)/goldstone-tai.yang
 	sysrepoctl -s $(GS_YANG_REPO) --install $(GS_YANG_REPO)/goldstone-sonic-interface.yang
+	sysrepoctl -s $(GS_YANG_REPO) --install $(GS_YANG_REPO)/sonic-common.yang
+	sysrepoctl -s $(GS_YANG_REPO) --install $(GS_YANG_REPO)/sonic-port.yang
+	sysrepoctl -s $(GS_YANG_REPO) --install $(GS_YANG_REPO)/sonic-interface.yang
```

Comment:
Can we add the right submodule which contains these SONiC YANG models and point them directly like what we're doing for OpenConfig models?

Reply:
Done